sycl/plugins/esimd_cpu/pi_esimd_cpu.hpp

```diff
@@ -23,6 +23,10 @@
 
 #include <malloc.h>
 
+namespace cm_support {
+#include <cm_rt.h>
+} // namespace cm_support
```

Comment:
Could you please clarify why this file is included in namespace?

Reply:
To make all symbols imported from CM prefixed with 'cm_support' name space.

Reply:
Comment inserted.